server/data/dev-data/answers.json

```diff
@@ -17,4 +17,31 @@
 		"upvotes": 35,
 		"downvotes": 4
 	}
-]
\ No newline at end of file
+]
+
```

Comment:
You have repeated the same code. Please remove the repetitive code and only add your answer here